aad/spring-cloud-azure-starter-active-directory/web-client-access-resource-server/aad-resource-server-obo/README.md

```diff
@@ -1,7 +1,7 @@
-# Azure OAuth 2.0 Sample for azure-spring-boot-sample-active-directory-resource-server-obo client library for Java
+# Azure OAuth 2.0 Sample for Azure AD Spring Boot Starter Resource Server Obo client library for Java
 
 ## Key concepts
-[Resource server access other resources usage] is an extension scenario of the *azure-spring-boot-sample-active-directory-resource-server* sample. Similarly, this sample illustrates how to protect a Java web API by restricting access to its resources to authorized accounts, and the restricted resource will access other restricted resource, such as Graph API and Custom API.
+[Resource server access other resources usage] is an extension scenario of the *aad-resource-server* sample. Similarly, this sample illustrates how to protect a Java web API by restricting access to its resources to authorized accounts, and the restricted resource will access other restricted resource, such as Graph API and Custom API.
```

Comment:
[One resource server accesses another resource server] is another usage scenario for the xxx lib.

The main point here is that we should explain what is "OBO" as the title denotes.

And, before we continue the sample, we need to list the infrastructures and its configurations. (and How-To, this is the prerequisites that have to be done before runing the app. ).  Along this, we explain the main concepts and demonstrate some diagrams just like the below section does.

Reply:
Good point, but I think it's not in this PR's scope, this pr is to fix some obvious errors in the sample project. @jialigit could you create the pr with the changes you propose here?